SecurityManager/Clients.Web/Classes/OrganizationalStructure/BocListInlineEditingConfigurator.cs

```diff
@@ -75,14 +75,14 @@ public virtual void Configure<TBusinessObject> (BocList bocList, Func<TBusinessO
           new BocMenuItem
           {
               ItemID = "New",
-              Text = GlobalResourcesHelper.GetString (_globalizationService, GlobalResources.New),
+              Text = WebString.CreateFromText (GlobalResourcesHelper.GetString (_globalizationService, GlobalResources.New)),
```

Comment:
```suggestion
              Text = WebString.CreateFromText (GlobalResourcesHelper.GetString (_globalizationService, GlobalResources.New)),
```
```suggestion
              Text = GlobalResourcesHelper.GetText (_globalizationService, GlobalResources.New),
```
We already have a utility for this since the WebButton branch.